ladon.go

```diff
@@ -48,6 +56,15 @@ func (l *Ladon) IsAllowed(r *Request) (err error) {
 // The IsAllowed interface should be preferred since it uses the manager directly. This is a lower level interface for when you don't want to use the ladon manager.
 func (l *Ladon) DoPoliciesAllow(r *Request, policies []Policy) (err error) {
 	var allowed = false
+	var deciders = Policies{}
+
+	defer func() {
+		if allowed {
+			l.auditLogger().LogGrantedAccessRequest(r, policies, deciders)
+		} else {
+			l.auditLogger().LogRejectedAccessRequest(r, policies, deciders)
```

Comment:
This will also trigger on error, is that intentional and if so, why?

Reply:
IMHO, it's just 3 lines of code that need adding in the corresponding return values on lines

* `return errors.WithStack(ErrRequestForcefullyDenied)`
* `return errors.WithStack(ErrRequestDenied)`
* `return nil`